tests/phpunit/class-wc-stripe-feature-flags-test.php

```
@@ -90,6 +90,79 @@ public function provide_test_is_oc_available() {
+		}
+
+		WC_Stripe_Remote_Config::reset_in_memory_cache();
+		delete_option( '_wcstripe_remote_config_live' );
+
+		// Force live mode so we hit the live cache.
+		$settings             = WC_Stripe_Helper::get_stripe_settings();
+		$settings['testmode'] = 'no';
+		WC_Stripe_Helper::update_main_stripe_settings( $settings );
+
+		if ( null !== $remote_value ) {
+			( new WC_Stripe_Remote_Config() )->apply(
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '119', column '10'). (undefined)

(MissingImport)